Undefined reference to ModelTest::ModelTest under defaults.
Clearly, we're not in a debug mode, but the sources think we are.

Patch is not upstream, made by SpaceToast on #alpine-devel
diff -ur a/client/ostinato.pro b/client/ostinato.pro
--- a/client/ostinato.pro	2019-05-28 09:32:50.000000000 -0400
+++ b/client/ostinato.pro	2019-06-04 19:14:23.477762179 -0400
@@ -123,6 +123,6 @@
 
 INCLUDEPATH += "../extra/modeltest"
 greaterThan(QT_MINOR_VERSION, 6) {
-CONFIG(debug, debug|release): LIBS += -L"../extra/modeltest/$(OBJECTS_DIR)/" -lmodeltest
-CONFIG(debug, debug|release): QT += testlib
+LIBS += -L"../extra/modeltest/$(OBJECTS_DIR)/" -lmodeltest
+QT += testlib
 }


